live-site/components/welcome-section/WelcomeSection.styles.ts

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components';
+import { max } from '../../../shared-ui/lib/responsive';
+import { colors } from '../../../shared-ui/style/colors';
+import { P } from '../../../shared-ui/style/typography';
+
+const StyledWelcomeSectionContainer = styled.div`
+  display: flex;
+  flex-direction: row;
```

Comment:
We can add margin: 1.3em here (see other comment for context on centering this entire section)